### PR TITLE
Added -lln[umeric-uid-gid] option

### PR DESCRIPTION
**Pull request by mouuff**
Added `-lln[umeric-uid-gid]`

This option does the same as ` -ll[s]`, but list **numeric** user and group IDs.

### CODE REVIEW

File: squashfs-tools/unsquashfs.c [hunk 1]
```diff
@@ -519,16 +520,17 @@ int print_filename(char *pathname, struct inode *inode)
...
+	struct group *group = NULL;
 	struct tm *t;
 
 	if(short_ls) {
 		printf("%s\n", pathname);
 		return 1;
 	}
 
-	user = getpwuid(inode->uid);
+  if (!numeric_uid_gid)
```
Comment (Dr-Emann) [inline squashfs-tools/unsquashfs.c]: This should be indented one more level.

File: squashfs-tools/unsquashfs.c [hunk 2]
```diff
@@ -2615,6 +2618,11 @@ int main(int argc, char *argv[])
 				strcmp(argv[i], "-ll") == 0) {
 			lsonly = TRUE;
 			short_ls = FALSE;
+		} else if(strcmp(argv[i], "-lln") == 0 ||
+				strcmp(argv[i], "-llnumeric-uid-gid") == 0) {
+			lsonly = TRUE;
+      numeric_uid_gid = TRUE;
```
Comment (Dr-Emann) [inline squashfs-tools/unsquashfs.c]: This needs more indentation.

File: squashfs-tools/unsquashfs.c [hunk 3]
```diff
@@ -2679,6 +2687,8 @@ int main(int argc, char *argv[])
 			ERROR("\t-ll[s]\t\t\tlist filesystem with file "
 				"attributes (like\n");
 			ERROR("\t\t\t\tls -l output), but don't unsquash\n");
+      ERROR("\t-lln[umeric_uid_gid]\tlike -ll[s], but list numeric user and group IDs "
+        "\n");
```
Comment (Dr-Emann) [inline squashfs-tools/unsquashfs.c]: The above line should be indented to match surrounding lines, and this line needs further indentation to match nearby continuation lines.

**Comment by plougher** [comment]
This pull request was never in a good enough state to merge.

I have implemented the option.

**Pull request closed without merging**